nsqadmin/static/js/lib/handlebars_helpers.js

```diff
@@ -0,0 +1,247 @@
+var $ = require('jquery');
+var _ = require('underscore');
+var Handlebars = require('hbsfy/runtime');
+
+var AppState = require('../app_state');
+
+var statsdPrefix = function(host, metricType) {
+    var prefix = AppState.get('STATSD_PREFIX');
+    var statsdHostKey = host.replace(/[\.:]/g, '_');
+    prefix = prefix.replace(/%s/g, statsdHostKey);
+    if (prefix.substring(prefix.length, 1) !== '.') {
+        prefix += '.';
+    }
+    if (AppState.get('USE_STATSD_PREFIXES') && metricType === 'counter') {
+        prefix += 'stats_counts.';
+    } else if (AppState.get('USE_STATSD_PREFIXES') && metricType === 'gauge') {
+        prefix += 'stats.gauges.';
+    }
+    return prefix;
+};
+
+var metricType = function(key) {
+    return {
+        'depth':                  'gauge', //eslint-disable-line key-spacing
+        'in_flight_count':        'gauge', //eslint-disable-line key-spacing
+        'deferred_count':         'gauge', //eslint-disable-line key-spacing
+        'requeue_count':          'counter', //eslint-disable-line key-spacing
+        'timeout_count':          'counter', //eslint-disable-line key-spacing
+        'message_count':          'counter', //eslint-disable-line key-spacing
+        'clients':                'gauge', //eslint-disable-line key-spacing
+        '*_bytes':                'gauge', //eslint-disable-line key-spacing
+        'gc_pause_*':             'gauge', //eslint-disable-line key-spacing
+        'gc_runs':                'counter', //eslint-disable-line key-spacing
+        'heap_objects':           'gauge', //eslint-disable-line key-spacing
+        'e2e_processing_latency': 'gauge' //eslint-disable-line key-spacing
```

Comment:
If it seems easier or cleaner, you could wrap these lines with:
```
/* eslint-disable key-spacing */
...
/* eslint-enable key-spacing */
```

http://eslint.org/docs/user-guide/configuring.html#configuring-rules